### ISSUE TITLE
WCAG 2.2 errata: success criteria 2.5.8

### ISSUE THREAD

**Issue by rscano** [w3c/wcag#3501]
there is an error in title for link "bounding box" (title="[New]"). Should be title="the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie."

**Comment by mbgower** [comment]
Since https://github.com/w3c/wcag/issues/3511 captures the underlying issue, and this is simply another example of the outcome, I am closing this as a duplicate.
